.github/workflows/reusable_stale.yml

```diff
@@ -0,0 +1,37 @@
+name: "Close stale issues & pull requests"
+on:
+  workflow_call:
+    inputs:
+      github-token:
+        required: true
+        type: string
+        description: "the repo's secrets.GITHUB_TOKEN"
+      days-before-close:
+        required: false
+        type: number
+        default: 7
+        description: "How long before a stale PR is closed (default 7 days)"
+      days-before-stale:
+        required: false
+        type: number
+        default: 30
+        description: "How long before a PR is marked as stale (default 30 days)"
+      exempt-pr-labels:
+        required: false
+        type: string
+        description: "Comma separated list of labels to ignore"
```

Comment:
do we want a value here?

Reply:
In doesn't, when accessed below it will be the default string value which is blank, but it won't error.

The reason I added defaults to the others was because we don't want to pass in the default number value of 0 for those.